src/Microsoft.PowerShell.Security/security/CatalogCommands.cs

```diff
@@ -89,26 +89,35 @@ protected override void ProcessRecord()
 
             Collection<string> paths = new Collection<string>();
 
+            bool _ShouldProcess = false;
```

Comment:
@PaulHigin Good catch!
@lzybkr If we take into consideration that cmdlets can work with large directories, then maybe we will remove the local variable and add a comment 'paths.Count > 0 is true only if ShouldProcess allow'?